test/command_line/test_find_bad_pixels.py

```diff
@@ -0,0 +1,25 @@
+import procrunner
+
+
+def test_find_bad_pixels(dials_data, tmp_path):
+
+    image_files = [
+        str(f) for f in sorted(dials_data("x4wide", pathlib=True).glob("*.cbf"))
```

Comment:
Does this _really_ need to test with all 90 images? This test takes ~40 seconds on my macbook pro - it would be nice to reduce this if possible to exercise the code with fewer images.

Reply:
Reasonable, will see if it ~ works with e.g. every 4th image

Reply:
Nope... and by the nature of it it finds thousands for narrow sweeps 🤔

Reply:
for the record: `image_files = sorted(...)` would also work, procrunner supports pathlib.Path arguments.

Reply:
Modified actual program to iterate over experiments, now test uses 20 not 90 images -> faster. Also fixed up "help"

```
Usage: dials.find_bad_pixels [options] (data_master.h5|data_00*.cbf)

Options:
  -h, --help            show this help message and exit
  -c, --show-config     Show the configuration parameters.
  -a ATTRIBUTES_LEVEL, --attributes-level=ATTRIBUTES_LEVEL
                        Set the attributes level for showing configuration
                        parameters
  -e EXPERT_LEVEL, --expert-level=EXPERT_LEVEL
                        Set the expert level for showing configuration
                        parameters
  -v                    Increase verbosity (can be specified multiple times)
  --phil=FILE           PHIL files to read. Pass '-' for STDIN. Can be
                        specified multiple times, but duplicates ignored.


Examples:

  dials.find_bad_pixels data_master.h5 [nproc=8]
```